### ISSUE TITLE
icmp-perf.rrd file not being created by poller

### ISSUE THREAD

**Issue by sarabveer** [librenms/librenms#15957]
### The problem

I am getting the following error when trying to view ICMP graphs:

<img width="500" alt="image" src="https://github.com/librenms/librenms/assets/4297171/fed9e0d5-0c09-4a5b-bd09-65c3a292b6b6">

I checked the poller in debug mode, and the poller is unable to create the `icmp-perf.rrd` file. (Poller debug log is included below).

I believe this issue is related to #15940.

### Output of ./validate.php

```txt
===========================================
Component | Version
--------- | -------
LibreNMS  | 24.3.0-25-g03e9b5229 (2024-04-19T09:03:34-04:00)
DB Schema | 2024_04_10_093513_remove_device_perf (291)
PHP       | 8.2.18
Python    | 3.11.2
Database  | MariaDB 10.11.6-MariaDB-0+deb12u1
RRDTool   | 1.7.2
SNMP      | 5.9.3
===========================================

[OK]    Composer Version: 2.7.2
[OK]    Dependencies up-to-date.
[OK]    Database connection successful
[OK]    Database Schema is current
[OK]    SQL Server meets minimum requirements
[OK]    lower_case_table_names is enabled
[OK]    MySQL engine is optimal
[OK]    Database and column collations are correct
[OK]    Database schema correct
[OK]    MySQL and PHP time match
[OK]    Active pollers found
[OK]    Dispatcher Service not detected
[OK]    Locks are functional
[OK]    Python poller wrapper is polling
[OK]    Redis is unavailable
[OK]    rrd_dir is writable
[OK]    rrdtool version ok
```

### Anything in the logs that might be useful for us?

```txt
SQL[update `devices` set `last_ping` = ?, `last_ping_timetaken` = ? where `device_id` = ? ["2024-04-19 12:33:20",0.036,4] 2.1ms] 
  
RRD[create /opt/librenms/rrd/localhost/icmp-perf.rrd --step 300 --source /opt/librenms/rrd/localhost/ping-perf.rrd DS:avg=ping[1]:GAUGE:600:0:65535 DS:xmt:GAUGE:600:0:65535 DS:rcv:GAUGE:600:0:65535 DS:min:GAUGE:600:0:65535 DS:max:GAUGE:600:0:65535   RRA:AVERAGE:0.5:1:2016 RRA:AVERAGE:0.5:6:1440 RRA:AVERAGE:0.5:24:1440 RRA:AVERAGE:0.5:288:1440 RRA:MIN:0.5:1:2016 RRA:MIN:0.5:6:1440 RRA:MIN:0.5:24:1440 RRA:MIN:0.5:288:1440 RRA:MAX:0.5:1:2016 RRA:MAX:0.5:6:1440 RRA:MAX:0.5:24:1440 RRA:MAX:0.5:288:1440 RRA:LAST:0.5:1:2016  -O]  
RRDtool Output: ERROR: error checking for source RRD /opt/librenms/rrd/localhost/ping-perf.rrd: No such file or directory
RRD[update /opt/librenms/rrd/localhost/icmp-perf.rrd N:0.036:5:5:0.022:0.041]  
RRDtool Output: ERROR: opening '/opt/librenms/rrd/localhost/icmp-perf.rrd': No such file or directory
SQL[select * from `device_outages` where `device_outages`.`device_id` = ? and `device_outages`.`device_id` is not null and `up_again` is null order by `going_down` desc limit 1 [4] 0.42ms]
```

**Comment by sarabveer** [comment]
Seems like the main issue here is the `--source` parameter in the `rrdtool create` command

**Comment by sarabveer** [comment]
Fixed in #15959
